### PR TITLE
Add bindings for quantitative POMDP analysis

### PR DESCRIPTION
**Pull request by AlexBork**
Introduction of bindings in accordance with the reworked quantitative POMDP analysis in Storm 1.8.
This is largely based on the work @TheGreatfpmK and @randriu did for the Storm-PAYNT POMDP integration (SAYNT).

### CODE REVIEW

**Comment by volkm** [comment]
Depends on https://github.com/moves-rwth/storm/pull/396

File: tests/pomdp/test_pomdp_quantitative_analysis.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        options = stormpy.pomdp.BeliefExplorationModelCheckerOptionsDouble(False, True)
+        options.use_state_elimination_cutoff = False
+        options.size_threshold_init = 10
+        options.use_clipping = False
+        belmc = stormpy.pomdp.BeliefExplorationModelCheckerDouble(model, options)
+        result = belmc.check(formulas[0].raw_formula, [])
+        print(result.lower_bound)
+        print(result.upper_bound)
+        print(result.induced_mc_from_scheduler)
+        print(result.cutoff_schedulers[0])
```
Comment (volkm) [inline tests/pomdp/test_pomdp_quantitative_analysis.py]: Can we replace the `print` statements by `assert`? Then we can actually test the correctness.

**Comment by TheGreatfpmK** [comment]
Hi everyone, can we work on getting this merged? https://github.com/moves-rwth/storm/pull/396 is already merged and we updated PAYNT to work with Storm 1.8 so it would be nice to have SAYNT fully operational with updated Storm/StormPy/PAYNT.

**Comment by sjunges** [comment]
@AlexBork can you resolve the remark by @volkm ? 

That should also retrigger the tests :-)

**Comment by AlexBork** [comment]
Sorry, I think I somehow missed this completely. I adjusted the tests and merged the latest state of main, so let's see if this works.

**Comment by sjunges** [comment]
Many thanks for your prompt response :-)

**Comment by volkm** [comment]
Thanks for integrating my comments.
The stable test currently fails. This is in principle not an issue, just increase the required Storm version. Looking at the failure it might be possible to fix the include though.

**Comment by AlexBork** [comment]
If I understand that correctly, the test runs using the stable branch of Storm. The include is already fixed in the latest Storm version, so isn't it necessary to release a new stable version containing the fix?  Or can I change the required Storm version to a certain commit? If so, can you give me a pointer how this is done?

**Comment by volkm** [comment]
Ah sorry, the error is in Storm and not stormpy. You need to increase the required Storm version [here](https://github.com/moves-rwth/stormpy/blob/master/setup.py#L16) and set it to 1.8.2.

**Comment by TheGreatfpmK** [comment]
Thanks everyone for a quick resolution of this!

**Pull request merged**